node_modules/oae-authentication/tests/test-external-strategies.js

```diff
@@ -985,6 +985,147 @@ describe('Authentication', function() {
         });
 
         /**
+         * Test that verifies that the user will be set to private if displayName resembles a shibboleth identifier
```

Comment:
As we're essentially using the same test 3 times, it'd be better to abstract a function that does the majority of the checks and we can just pass a remote user into. We can then execute it 4 times (for the `htttp://` case as well).